### ISSUE TITLE
Test XMLBear on Appveyor

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-bears#1354]
XMLBear is skipped on the Appveyor Windows builds.
There are dependencies that need to be installed/configured.

**Comment by Techievena** [comment]
`lxml version==3.6.0` has binaries built for windows. Can fix dependency issues in https://github.com/coala/coala-bears/pull/1337 and https://github.com/coala/coala-bears/pull/1364 Should I commit this part first and then work on Testing XMLBear on Appveyor

**Comment by jayvdb** [comment]
> Should I commit this part first and then work on Testing XMLBear on Appveyor?

Installing lxml on Windows is a tiny part of the problem.   It can be a separate commit, but it wont be merged until one of the bears is working.

**Comment by Shade5** [comment]
https://github.com/coala/coala-bears/pull/1337 cannot be merged until this id fixed :cry:

**Comment by jayvdb** [comment]
@shade5, do you know how to do it?

@techievena, if you havent figured it out, should I do it, since it is holding everyone up?

**Comment by Techievena** [comment]
@jayvdb My exams are going on right on and I am not getting enough time right now. It's better you unassign me and do it, if it is holding up others.

**Comment by Shade5** [comment]
Nope, I tried to fix it in my pr, but couldn't figure out how to install the dependencies in windows

**Comment by jayvdb** [comment]
On Appveyor, you can fetch libxml2, libxslt, zlib and libiconv packages using nuget.
